src/router/routes.js

```diff
@@ -184,6 +184,19 @@ export default [
 		],
 
 	},
+	{
+		path: '/settings/security',
+		component: () => import('@/pages/Settings/SecuritySettings'),
+		children: [{
+			path: '',
+			name: 'password',
+			component: () => import('@/pages/Settings/SecuritySettingsPage'),
+			meta: {
+				activeLoginRequired: true,
+			}
+		},
+		]
+	},
 	{ path: '/settings/security/mfa', component: () => import('@/pages/Settings/TwoStepVerifyPage') },
```

Comment:
You can make this a child of the settings/security route above

Reply:
The TwoStepVerifyPage has it's own header and footer and putting this as a child of settings/security results in: 
<img width="1217" alt="Screen Shot 2020-10-19 at 3 50 10 PM" src="https://user-images.githubusercontent.com/1521381/96520085-25aa3880-1223-11eb-8c84-310c45ce6a6d.png">

Reply:
Yeah you're right, I'm fine with leaving it as is then